src/util/TypeUtils.ts

```diff
@@ -0,0 +1,4 @@
+
+export interface Indexed<T> {
```

Comment:
@Eirmas here Nath is u sing your <> Typescript syntax.
Nath can you briefly describe what this <> is doing?

Reply:
That interface is an alias for an indexed type whose properties are of type T. If you're not familiar with generics, have a look here https://www.typescriptlang.org/docs/handbook/generics.html

It's a difficult concept, but very powerful